src/Sulu/Bundle/PreviewBundle/Command/CacheClearCommand.php

```diff
@@ -28,6 +28,8 @@ class CacheClearCommand extends BaseCacheClearCommand
     protected function configure()
     {
         parent::configure();
+
+        $this->setName('cache:clear');
```

Comment:
this is needed because of the "lazy initialized" command `Symfony\Bundle\FrameworkBundle\Command\CacheClearCommand` dont have a name